aboutstyles.css

```diff
@@ -0,0 +1,397 @@
+  padding: 0;
+  margin-top: 15px;
+  border: #272a31 1px solid;
+  align-self: center;
+  text-align: center;
+  width: 80%;
+}
+
+.logo-specifics {
+  width: 250px;
+  height: 250px;
+}
+
+.pastproject-section {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.summit22 {
+  align-self: center;
+  width: 320px;
+  height: 250px;
+}
+
+.summit20 {
+  align-self: center;
+  margin-top: 10px;
+  width: 320px;
+  height: 250px;
+}
+
+.partner-section {
+  background-color: rgb(39, 42, 50);
+  height: 550px;
+}
+
+.partner-heading {
+  color: #fff;
+}
+
+.partner-div {
+  text-align: center;
+}
+
+.companies {
+  width: 150px;
+  height: 150px;
+}
+
+.footer-div {
+  text-align: center;
+}
+
+.footer-logo {
+  width: 100px;
+  height: 100px;
+}
+
+/* ========Media Queries=======1 */
+@media screen and (min-width: 768px) {
+  nav.nav-menu-about {
+    display: none !important;
+  }
+
+  header .mobile-menu {
+    display: none;
+  }
+
+  nav.nav-menu-desktop-about {
+    display: flex;
+    background-color: #272a31;
+    justify-content: flex-end;
+    padding-right: 54px;
+  }
+
+  .list-about {
+    display: flex;
+    color: #fff;
+    list-style-type: none;
+  }
+
+  .list-about li {
+    padding-right: 44px;
+  }
+
+  .header-icon-about {
+    text-decoration: none;
+    color: #fff;
+  }
+
+  nav.nav-menu-desktop2-about {
+    display: flex;
+  }
+
+  .logolink-about {
+    width: 90px;
+    height: 90px;
+  }
+
+  .list2-about {
+    display: flex;
+    padding: 0;
+    margin: 0;
+    list-style-type: none;
+    height: 60px;
+  }
+
+  .list2-about li {
+    padding: 25px;
+    align-self: center;
+  }
+
+  .list2-about li a {
+    text-decoration: none;
+    color: #272a31;
+  }
+
+  .summit-logo-about {
+    position: relative;
+    left: 80px;
+  }
+
+  .nav-links-about {
+    position: relative;
+    left: 465px;
+  }
+
+  article .headline-about {
+    display: none;
+  }
+
+  article .headlinedesktop-about {
+    display: block;
+  }
+
+  .logo-section {
+    background-color: #d3d3d3;
+  }
+
+  .logo-div {
+    width: 40%;
+  }
+
+  .summit22 {
+    width: 420px;
+    height: 350px;
+  }
+
+  .summit20 {
+    margin-top: 10px;
+    float: left;
+    width: 420px;
+    height: 350px;
+  }
+
+  .partner-section {
+    height: 300px;
+  }
+
+  .partner-div {
+    text-align: center;
+  }
+}
+
+/* ========Desktop version CSS Rule set=======1 */
+.topdesktop-about {
+  background-image: url('./images/topbg.png');
+  background-size: cover;
+  height: 200px;
+}
+
+.topheadingdesktop-about {
+  color: orangered;
+  font-size: 34px;
+  font-weight: 400;
+  position: relative;
+  top: 150px;
+  left: 250px;
+}
+
+.topheadingdesktopdiv-about {
+  background-image: url('./images/bgimagedesign.png');
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position: 350px 0;
+  background-color: #fff;
+  position: relative;
+  left: 200px;
+  padding: 45px;
+  width: 100%;
+  height: 545px;
+}
```

Comment:
- [ ] Kindly make sure all text in this container is aligned in the center in order to match the provided design as much as possible.

- [ ] Kindly make sure the `email` is underlined as the provided design.

    - ### YOUR DESIGN
     ![image](https://user-images.githubusercontent.com/87197412/219405555-a90bb15d-4401-4c67-87a6-0599f45fd574.png)

    - ### FIGMA DESIGN
    
     ![image](https://user-images.githubusercontent.com/87197412/219405709-eab422be-b261-4928-848a-067a4c9d3546.png)

Reply:
Suggested changes made also changed background color of that section and underlined the email as well as discussed. Screen shot attached below for your kind reference.
![image](https://user-images.githubusercontent.com/117091923/219502902-eaad10b5-b67d-4812-9e16-bce8edf04fd7.png)